doc/ch-image_desc.rst

```diff
@@ -139,8 +139,10 @@ In descending order of priority, this directory is located at:
   :code:`$CH_IMAGE_STORAGE`
     Environment variable.
 
-  :code:`/var/tmp/$USER/ch-image`
-    Default.
+  :code:`/var/tmp/$USER.ch`
+    Default. (Previously, the default was :code:`/var/tmp/$USER/ch-image`. If
+    a valid storage directory is found at the old default path,
+    :code:`ch-image` tries to move it to the new default path.)
```

Comment:
Do you anticipate Charliecloud using `/var/tmp/$USER.ch` for operations besides those driven by `ch-image`? If so, any thoughts on having `/var/tmp/$USER.ch/ch-image` be the default?

Reply:
Yes. In fact we already have `/var/tmp/$USER.ch/mnt` for the default SquashFS. The thinking is that adding another directory level makes maintenance more complicated because `mkdir(2)`/`rmdir(2)` aren't recursive, and the recursive options (analogous to `mkdir -p` or `rm -Rf`) can be more high powered than needed, and this complexity isn't outweighed by clear benefits.